kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue

```diff
@@ -248,7 +249,7 @@
         showAddAddressModal,
         facilityDisabled,
         handleAddedAddress,
-        to_continue,
+        toContinue,
       };
     },
     inject: ['changeFacilityService'],
```

Comment:
We could return the changeFacilityService from `setup` and not doubly inject - but not a blocker.